### PR TITLE
Feature/add view fav btn

### PR DESCRIPTION
**Pull request by Eva-Navarrete89**
# Description

-  Added Display favorite page functionality
-  Added saved recipe's page

Fixes # (issue)

- Fixed display favorite recipe 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

- [x] open index.html
- [x] console.log

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Comment by Eva-Navarrete89** [comment]
@tarynmartin I made the favorites recipe page and display recipe on the page function.

File: main.js [hunk 1]
```diff
@@ -37,25 +39,24 @@ var currentDish;
 
 function displayFood(event) {
   event.preventDefault();
-  // fidure out a way to clear the p tag everytime lets cook button is selected.
- for (var i = 0; i < radioBtns.length; i++) {
-   console.log(radioBtns[i]);
-    if(radioBtns[i].checked && radioBtns[i].value ==='sides') {
+  for (var i = 0; i < radioBtns.length; i++) {
+    console.log(radioBtns[i]);
```
Comment (tarynmartin) [inline main.js:43]: make sure you remove your console logs!

File: main.js [hunk 2]
```diff
@@ -73,18 +74,30 @@ function getDish(currentDish) {
 var favoriteDish = [];
```
Comment (tarynmartin) [inline main.js:74]: I would put this near to your other variables; try and group like things together.

**Review by tarynmartin — APPROVED** [review]
Looks good! Checking the radio buttons function looks great and the function for checking for saved recipes & the one to display them look good. Make sure you remove all console logs when you no longer need them and remove unnecessary comments

**Pull request merged**